addons/osfstorage/views.py

```diff
@@ -105,7 +105,7 @@ def osfstorage_get_revisions(file_node, payload, target, **kwargs):
 
     version_count = file_node.versions.count()
     # Don't worry. The only % at the end of the LIKE clause, the index is still used
```

Comment:
Does this comment still apply? Can't quite tell!

Reply:
I don't think it's relevant anymore!